lib/node_modules/@stdlib/blas/ext/base/dcusumkbn/src/dcusumkbn.c

```diff
@@ -38,7 +38,7 @@
 * @param Y        output array
 * @param strideY  Y stride length
 */
-void stdlib_strided_dcusumkbn( const int64_t N, const double sum, const double *X, const int64_t strideX, double *Y, const int64_t strideY ) {
+double stdlib_strided_dcusumkbn( const int64_t N, const double sum, const double *X, const int64_t strideX, double *Y, const int64_t strideY ) {
```

Comment:
This change in return type should be reverted.

```suggestion
void stdlib_strided_dcusumkbn( const int64_t N, const double sum, const double *X, const int64_t strideX, double *Y, const int64_t strideY ) {
```